hwcomposer/qeglfsscreen.cpp

```diff
@@ -63,6 +63,7 @@ QEglFSScreen::QEglFSScreen(HwComposerContext *hwc, EGLDisplay dpy)
     , m_screenOrientation(Qt::PrimaryOrientation)
     , m_orientationSensor(new QOrientationSensor(this))
 #endif
+    , m_powerState(PowerStateOn)
```

Comment:
Wondering should this actually do setPowerState() to ensure the state is consistently on.